src/app/icons/ArrowIcon.tsx

```diff
@@ -22,7 +22,7 @@ const ArrowIcon: FC<ArrowProps> = ({ arrowDirection = ArrowDirection.UP }) => {
   return (
     <ArrowSvgIcon
       arrowDirection={arrowDirection}
-      width="8"
+      width="13"
```

Comment:
Is icon positioned correctly for all directions when svg `viewBox` is not in sync with this change?

Reply:
Yep, at least on Chrome and Firefox

| Before | After |
| --- | --- |
|  ![localhost_1234_ (1)](https://user-images.githubusercontent.com/3758846/211575268-0859d3fe-df6a-4deb-bcd3-587095d57331.png) |  ![localhost_1234_](https://user-images.githubusercontent.com/3758846/211575264-07a9175c-5d11-4fa7-9a70-010aa022ca5e.png)  |